src/js/bootstrap-slider.js

```diff
@@ -740,8 +749,15 @@ const windowIsDefined = (typeof window === "object");
 				this.hideTooltip = this._hideTooltip.bind(this);
 
 				if (this.options.ticks_tooltip) {
-					this.createTickMouseOverListener(this, this.handle1);
-					this.createTickMouseOverListener(this, this.handle2);
+					var callbackHandle = this._addTickListener();
+                    var mouseEnter = callbackHandle.addMouseEnter(this, this.handle1);
+                    var mouseLeave = callbackHandle.addMouseLeave(this, this.handle1);
+                    callbackHandle.addMouseEnter(this, this.handle2);
+                    callbackHandle.addMouseLeave(this, this.handle2);
+                    this.handleCallbackMap = {
```

Comment:
likewise, need to map each handle to its event listeners.

I would recommend something like the following:

```js
var callbackHandle = this._addTickListener();

// generate event listeners for handle1 and store in map
var mouseEnter = callbackHandle.addMouseEnter(this, this.handle1);
var mouseLeave = callbackHandle.addMouseLeave(this, this.handle1);

this.handleCallbackMap["handle1"] = {
  mouseEnter: mouseEnter,
  mouseLeave: mouseLeave
};

// generate event listeners for handle2 and store in map
mouseEnter = callbackHandle.addMouseEnter(this, this.handle2);
mouseLeave = callbackHandle.addMouseLeave(this, this.handle2);

this.handleCallbackMap["handle2"] = {
  mouseEnter: mouseEnter,
  mouseLeave: mouseLeave
};
```